### PR TITLE
Forrr new params consent

### PR DESCRIPTION
**Pull request by mobimation-gunnar**
Implements module scope sharing of new parameters "jobRole" and "companyName".
af-connect-module scans files in module for occurrences of value assignments to
data-job_title
or
data-company_name 
in HTML elements of class "af-connect-module".

Such as :
``` 
<div id="af-connect-button" class="af-connect-module"
      data-job_title="Developer"
      data-company_name="Visma"
</div>
```
This enables use of these values in rendering.

### CODE REVIEW

**Review by ixuz — CHANGES_REQUESTED** [review]
Please update README.md to include descriptions for:
- data-company_name
- data-job_title

Also update the version number in README.md to `1.2.0-beta`

**Comment by mobimation-gunnar** [comment]
Change done, will push update

File: README.md [hunk 1]
```diff
@@ -146,43 +144,56 @@ Find all available configuration properties in section: [Configuration](#configu
...
+In the process of requesting and retrieving user profile data from AF there is a step where the user gives consent for sharing the data to a third party.
+In order for the consent form to be formulated properly three parameters are provided where you can specify key parameters about the data:
+
+There are three parameters provided for customizing the data sharing consent form.
 
+| Custom property   | Description                |
+| ----------------- | -------------------------- |
+| data-purpose      | Phrase of intended purpose |
+| data-company_name | Prospective employer       |
+| data-job_role     | The job role applied for   |
```
Comment (ixuz) [inline README.md:156]: Slight change request `data-job_role`, should be `data-job_title`.

**Review by ixuz — CHANGES_REQUESTED** [review]
Just a small change request, and then I think we're good to go.

**Comment by mobimation-gunnar** [comment]
Just det var fel.  Ok jag ändrar nu.

File: README.md (hunk 1 above)
Comment (ixuz) [reply README.md]: Resolved by https://github.com/MagnumOpuses/af-connect-module/pull/49/commits/49d001c12ab037b03d1ba682ea1130f5fee44c55

**Review by ixuz — APPROVED** [review]
Looks good, you may now squash and merge

**Pull request merged**